frontend/badges/cppCourseGraduate/query.sql

```diff
@@ -0,0 +1,46 @@
+SELECT
+   u.user_id
+from
+	Users u
+INNER JOIN (
+    SELECT
+        i.user_id,
+        MAX(r.contest_score) AS scoreMax,
```

Comment:
```suggestion
        MAX(r.contest_score) AS max_contest_score,
```

esto estaba bien antes. hay que cuidar que las convenciones de nombres sean consistentes dentro del contexto: todos los nombres de variables y columnas en SQL van en `snake_case`.